packages/analytics-js-plugins/src/types/common.ts

```diff
@@ -1,10 +1,19 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import { IApplicationState } from '@rudderstack/analytics-js/state/IApplicationState';
+
+export type { RudderEvent } from '@rudderstack/analytics-js/components/eventManager/types';
+export type { QueueOpts } from '@rudderstack/analytics-js/state/types';
+export type { ILogger } from '@rudderstack/analytics-js/services/Logger/types';
```

Comment:
in the future we will gather all comon code that needs to be shared between the sdk and the plugins in a separate package. This is more of a reminder that these should be moved there.

Also in our plugins code we import from common file instead of the sdk files directly so it is easier to refactor when this happens